packages/atoms/src/ListItem/ListItemIcon.tsx

```diff
@@ -0,0 +1,36 @@
+import { View } from 'react-native';
+import { Icon } from '../Icon/Icon';
+import React from 'react';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { IconType } from '../Icon/iconTypes';
+
+type ListItemIconProps = {
+    icon: IconType;
+};
+
+const listItemIconContainerStyle = prepareNativeStyle(() => ({
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginRight: 16,
+}));
+
+const listItemIconStyle = prepareNativeStyle(utils => ({
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: 48,
+    width: 48,
+    borderRadius: utils.borders.radii.round,
+    backgroundColor: utils.colors.gray200,
+}));
+
+export const ListItemIcon = ({ icon }: ListItemIconProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <View style={applyStyle(listItemIconContainerStyle)}>
+            <View style={applyStyle(listItemIconStyle)}>
+                <Icon type={icon} />
+            </View>
+        </View>
```

Comment:
```suggestion
        <View style={applyStyle(listItemIconContainerStyle)}>
            <View style={applyStyle(listItemIconStyle)}>
                {children}
            </View>
        </View>
```

This seems to me like it is just a wrapper for icon. I would just have this accept children and wrap the Icon with ListItemIconWrapper. But maybe you could just create new IconSize / RoundIcon cmp that will be rounded + have gray background since this exact thing will be done on slide down modal and would be nice to have this wrapper more reusable.

Reply:
This is probably connected with https://github.com/orgs/trezor/projects/61/views/1

Reply:
Well that's why I was commenting. Children is unnecessary, you're right. But we will always put Icon inside so basically it's an Icon wrapper but in this case, since it's going to be a component that is not used just in ListItem (modal as well for example), I would move it from ListItem and make the naming more generic. Maybe even move it to Icon folder since this is more Icon related than ListItem related. 

I don't feel very good about importing ListItemIcon to Modal, that just doesn't make sense to me.

Reply:
I agree with Bohdan. At least for the closing circle in the modal, it would be better to have that icon "wrapped".

![Snímek obrazovky 2022-05-27 v 9 47 15](https://user-images.githubusercontent.com/2227592/170654651-2a1f9aaf-ce84-4c42-bcf5-5f33ac63f67a.png)

Reply:
Keep it simple, better have two small components than one super universal.

Reply:
It's not worth it, creating universal component for saving like 5 lines of styles used only on two places and you even need to make it conditional because dimensions are slightly different. 

Universal components for this "nearly" identical parts, but with only very specific places where it can be used, are hell to maintain because with every change you need to check every slightly different conditional variant in place of usage etc. It's definitely not worth for saving few lines. 

Creating CloseIcon makes more sense than IconWrapper, but I think it's not worth now, because maybe we will decide to add for example size prop to IconButton #5357 and it can work as close button. But also pay attention that it's not worth it to adding some conditions to this component because of one specific usage like in input clear icon. It's worth only when it's that variant will be used multiple times in many places.